common/src/main/java/com/viaversion/viaversion/rewriter/ItemRewriter.java

```diff
@@ -36,14 +36,14 @@
 
 public class ItemRewriter<C extends ClientboundPacketType, S extends ServerboundPacketType,
     T extends Protocol<C, ?, ?, S>> extends RewriterBase<T> implements com.viaversion.viaversion.api.rewriter.ItemRewriter<T> {
-    private final Type<Item> itemType;
-    private final Type<Item> mappedItemType;
-    private final Type<Item[]> itemArrayType;
-    private final Type<Item[]> mappedItemArrayType;
-    private final Type<Item> itemCostType;
-    private final Type<Item> mappedItemCostType;
-    private final Type<Item> optionalItemCostType;
-    private final Type<Item> mappedOptionalItemCostType;
+    protected final Type<Item> itemType;
```

Comment:
No need to make them protected, the important one have getter functions at the bottom which should be used.